pallets/parachain-staking/src/types.rs

```diff
@@ -1566,3 +1567,39 @@ pub enum BondAdjust<Balance> {
     Increase(Balance),
     Decrease,
 }
+
+// Data structure for tracking collator rewards
+#[derive(Clone, PartialEq, Eq, Encode, Decode, RuntimeDebug, Default, TypeInfo)]
+pub struct CollatorPayoutInfo<Balance> {
+    pub total_stake_accumulator: CollatorPayoutPeriodIndex,
```

Comment:
I'd suggest some different naming here.
```suggestion
    pub number_of_accumulations: CollatorPayoutPeriodIndex,
```